lua/tardis/interiors/modules/sh_exteriors.lua

```diff
@@ -0,0 +1,113 @@
+function TARDIS:SetExteriorCategory(ext_id, category)
+    for k,v in pairs(self.ExteriorCategories) do
+        if v[ext_id] then
+            v[ext_id] = nil
+        end
+    end
+    self.ExteriorCategories[category] = self.ExteriorCategories[category] or {}
+    self.ExteriorCategories[category][ext_id] = true
+end
+
+function TARDIS:ClearExteriorMetadata(id)
+    self.ExteriorsMetadata[id] = nil
+    for k,v in pairs(self.ExteriorsMetadataRaw) do
+        if v.Base == id then
+            self:ClearExteriorMetadata(k)
+        end
+    end
+end
+
+function TARDIS:AddExterior(t)
+    local id = t.ID
+    t.Category = t.Category or "Miscellaneous"
```

Comment:
`Miscellaneous` should be using the language system

Reply:
fixed in d317f2296d75d84c887c860916cd3a3ca1251e25 and f3b14d8307a669aa36e9efbc8fb124f77227d5ae